### PR TITLE
Update Docs: Getting Started and Index

### PR DESCRIPTION
**Pull request by mdahlman**
### Brief summary of the change made

The Getting Started samples did not work correctly for a brand new user. It was an effect of the recent change making dialect mandatory. (README.md was updated for this change, but index.rst and gettingstarted.rst had not been updated. Further, some of the output has changed (because of rule changes). I updated the output to reflect this.

Per Barry P's comment last time, I did not create an issue ahead of time.

### Are there any other side effects of this change that we should be aware of?
None that I know of.

### Pull Request checklist
- [x] Please confirm you have completed any of the necessary steps below.

- Included test cases to demonstrate any code changes, which may be one or more of the following:
  - `.yml` rule test cases in `test/fixtures/rules/std_rule_cases`.
  - `.sql`/`.yml` parser test cases in `test/fixtures/dialects` (note YML files can be auto generated with `tox -e generate-fixture-yml`).
  - Full autofix test cases in `test/fixtures/linter/autofix`.
  - Other.
- Added appropriate documentation for the change.
- Created GitHub issues for any relevant followup/future enhancements if appropriate.

### CODE REVIEW

**Review by tunetheweb — APPROVED** [review]
Thanks!

File: docs/source/gettingstarted.rst [hunk 1]
```diff
@@ -76,22 +76,21 @@ same folder that you're currently in with the following content:
     SELECT a+b  AS foo,
     c AS bar from my_table
 
-You can then run :code:`sqlfluff lint test.sql` to lint this file.
+You can then run :code:`sqlfluff lint test.sql --dialect ansi` to lint this file.
```
Comment (tunetheweb) [inline docs/source/gettingstarted.rst:79]:
```suggestion
You can then run :code:`sqlfluff lint test.sql --dialect ansi` to lint this
file.
```

Comment (mdahlman) [reply docs/source/gettingstarted.rst]: Does this imply that my environment (VSCode) wasn't set to wrap at 80 characters?

Comment (tunetheweb) [reply docs/source/gettingstarted.rst]: Wasn't aware you could set line limits in VSCode.

**Pull request merged**